plugins/fields.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fields
+spec:
+  platforms:
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.1.1-beta/kubectl-fields-Darwin-x86_64.tar.gz
+    sha256: ac82ee062e8bf1a9c6f6fab2e718e6d560c4ca1855616c2a67b0411792b575bc
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.1.1-beta/kubectl-fields-Linux-x86_64.tar.gz
+    sha256: 051140e9df85e7c3b982fc7b4199084441bbfc6328624f9a0dff07193058f135
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.1.1-beta/kubectl-fields-Windows-x86_64.zip
+    sha256: d188b7d2d1c567d7b074a3d8eb6c075aa8713e483074b839ff0da0300a8c710a
+    bin: kubectl-fields.exe
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: v1.1.1-beta
+  homepage: https://github.com/rewanth1997/kubectl-fields
+  caveats: |
+    Usage:
+      $ kubectl fields <RESOURCE> [options] <PATTERN>
+
+    Options:
+      -h, --help          help for kubectl-fields
+      -i, --ignore-case   Ignore case distinction
+      --stdin         Expects input via pipes
+
+    Read the documentation:
+      https://github.com/rewanth1997/kubectl-fields/tree/v1.0.1
+  shortDescription: Grep resources hierarchy by field name
+  description: |
```

Comment:
Can you wrap the description at 80 characters, so it is nicely displayed on smaller terminals?

Reply:
```
kubectl-fields parses specified kubectl resources to match given pattern(s).
It prints matched fields parental hierarchy in one-liner format.
```

I guess, this kind of formatting should work well